packages/InteractiveProfilingTool-Tests.package/IPTTreeModelTestCase.class/instance/testSourceCodeWhenCurrentSelectionIsNil.st

```diff
@@ -0,0 +1,11 @@
+as yet unclassified
+testSourceCodeWhenCurrentSelectionIsNil
+
+	| tree |
+		
+	tree := IPTTreeModelMock new.
+	tree currentSelection: MessageTallyMock new.
+	
+	tree currentSelection: nil.
+	
+	self assert: nil equals: tree selectedMessage.
```

Comment:
Please consider rephrasing this line to
```smalltalk
self assert: tree selectedMessage isNil
```
Note that you can get rid of the dot `.` at the end of the line.